app/components/deprecation-item.js

```diff
@@ -0,0 +1,26 @@
+import Ember from "ember";
+const { Component, computed } = Ember;
+const { notEmpty } = computed;
+
+export default Component.extend({
+  isExpanded: true,
+
+  tagName: '',
+
+  hasMap: notEmpty('model.hasSourceMap'),
+
+  actions: {
+    toggleExpand() {
+      this.toggleProperty('isExpanded');
+    },
+    openResource(...args) {
+      this.sendAction('openResource', ...args);
```

Comment:
I mentioned this below, but if we pass closure actions in here we don't need to manually bubble these actions like this.